methods/method.go

```diff
@@ -1,10 +1,25 @@
 package methods
 
-import "github.com/prometheus/prometheus/pkg/rulefmt"
+import (
+	"time"
+
+	"github.com/prometheus/common/model"
+	"github.com/prometheus/prometheus/pkg/rulefmt"
+)
+
+type Window struct {
+	Duration     model.Duration `yaml:"duration"`
+	Consumption  float64        `yaml:"consumption"`
+	Notification string         `yaml:"notification"`
+}
 
 type AlertErrorOptions struct {
 	ServiceName        string
 	AvailabilityTarget float64
+	SLOWindow          time.Duration
+
+	Windows     []Window
+	ShortWindow bool
```

Comment:
@pedrokiefer I have a question about ShortWindow, will be false by default ?

Reply:
That was the idea... 

It could actually be a `float64` and let the user choose his ShortWindow fraction. Using `12.0` defaults to a 5min window being used on a SLO of 1h. No value, no ShortWindow will be created.

Reply:
I changed to true by default to wont have break changes.